### PR TITLE
Add 2 mentors to the mentor section

### PR DESCRIPTION
**Pull request by gaeun**
This is for the GCI task: https://codein.withgoogle.com/dashboard/task-instances/4853271573299200/.
Here's a preview: https://gaeun.github.io/gci16.fossasia.org/

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -45,3 +45,19 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
+  
+- name: Damini Satya
+  github: daminisatya
+  image: daminisatya.jpg
+  blog:
+  twitter: Daminisatya
+  facebook:
```
Comment (hemantjadon) [inline _data/mentors.yml:54]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/_data/mentors.yml
+++ b/_data/mentors.yml
@@ -51,7 +51,7 @@
   image: daminisatya.jpg
   blog:
   twitter: Daminisatya
-  facebook: 
+  facebook:
   bio:
 
 - name: Nicco Kunzmann
```

File: _data/mentors.yml [hunk 2]
```diff
@@ -45,3 +45,19 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
+
```
Comment (hemantjadon) [inline _data/mentors.yml:48]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/_data/mentors.yml
+++ b/_data/mentors.yml
@@ -45,7 +45,7 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
-  
+
 - name: Damini Satya
   github: daminisatya
   image: daminisatya.jpg
```

**Comment by gaeun** [comment]
@hemantjadon Sorry, I'm not sure what you mean by this. It seems like I followed the formatting of the file without adding whitespace?

**Comment by hemantjadon** [comment]
Hey I know there has been some issue.
We are trying to use gitmate to help you to in running proper build. But there are some issues like It is using my personal account, and send you suggestions.

Sorry for that inconvinience.

You can close this PR and open up a new one. also please squash the commits, also check for merge conflicts.

**Pull request closed without merging**